apps/okreads-e2e/src/integration/reading-list.spec.ts

```diff
@@ -11,4 +11,68 @@ describe('When: I use the reading list feature', () => {
       'My Reading List'
     );
   });
+
+  describe('When: I click to add item to reading list', () => {
+    function addItemToReadingList() {
+      cy.get('input[type="search"]').type('javascript');
+      cy.get('form').submit();
+      cy.get('.book--content--info button').eq(0).click();
+    }
+    function clearReadingList() {
+      cy.get('[data-testing="toggle-reading-list"]').click();
+      cy.get('.reading-list-item button').click({multiple: true});
+      cy.get('[data-testing="reading-list-container"] button').click();
+    }
+
+    beforeEach(() => {
+      cy.startAt('/');
+      
+      const readingListCount = cy.$$('tmo-total-count span.mat-badge-content');
+      if ( readingListCount.length !== 0 ) {
+        clearReadingList();
+      }
+    });
+
+    it('Then: I should be able to undo my action', () => {
+      addItemToReadingList();
+      const snackBar = cy.get('simple-snack-bar button').eq(-1);
```

Comment:
You can't really do this in Cypress. It may work, but it's probably not doing what you expect.

cf. https://docs.cypress.io/guides/core-concepts/variables-and-aliases#Return-Values

Reply:
updated